app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -35,7 +34,10 @@
     isPertaxUrl:Boolean
 )(implicit request: Request[_], user: NispUser, partialRetriever: CachedStaticHtmlPartialRetriever)
 
-
+@pageScripts = {
```

Comment:
One thing I just realised, we can only call the 'pageScripts' element once, and we are already using it in the NI record page - it means we can either call it on the main page or on each individual page. 
Does everybody agree with adding the nirecord.js script in the main too ?